documentation/.markdownlint.jsonc

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
What would the impact be if we also used this configuration for the entire project? Might not be bad to adhere to the same standards everywhere.

Reply:
I thought to separate them because all non documentation markdown files get rendered inside GitHub... the 4 vs 2 spaces multi list thing and all that.

I was going to look into extending from the base config though, that apparently is possible, so there is less duplication.

Reply:
Apparently, the 4 spaces multilist gets rendered as it should when viewing through GitHub so I guess we can use the same config throughout the repository if we want.

I've added it with extension for now but either is fine.

Reply:
What exactly are the required differences? If it's only the 4 vs 2 spaces we could just enforce that over the entire repository (since technically that is the correct way to do it).

Reply:
Didn't see your 2nd comment when I commented. So yes, would look into using the same config everywhere if possible.